x-pack/filebeat/module/zeek/README.md

```diff
@@ -0,0 +1,73 @@
+# Zeek (Bro) module
+
+## Caveats
+
+* Module is to be considered _alpha_.
+* Field names will be changing for 7.0 to comply with Elastic Common Schema (ECS).
+
+## How to try the module from source
+
+Install Zeek/Bro (for MacOS with Brew)
+
+```
+brew install bro
+```
+
+Configure it to process network traffic and generate logs. 
```

Comment:
I think this will look better as a list, i.e. start each line with *.

Reply:
Yes, I agree. Will update that